.github/workflows/build.yml

```diff
@@ -8,8 +8,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        options: [ '-mb braces', '-mc', '--toplevel -mc passes=10,pure_getters,unsafe' ]
-        script: [ acorn.sh, buble.sh, butternut.sh, mathjs.sh, rollup-es.sh, rollup-ts.sh, sucrase.sh ]
+        options: [ '-mb braces', '--ie8 -c', '-mc', '--toplevel -mc passes=10,pure_getters,unsafe' ]
```

Comment:
`passes=10` is probably excessive - `web-tooling-benchmark` alone can take 15 minutes on my machine with 10 passes.

What do you think about this?
```
options: [ '-mb braces', '--ie8 -c', '-mc', '--toplevel --no-rename -mc passes=3,pure_getters,unsafe' ]
```

Reply:
`passes=3` SGTM

Both `-mb braces` & `--ie8 -c` imply `--no-rename`, so I think we should let the last one explore all inlining opportunities (`inline` does not intrinsically resolve conflicting variable names)